slowfast/config/defaults.py

```diff
@@ -92,6 +93,17 @@
 _C.TEST.CHECKPOINT_TYPE = "pytorch"
 
 
+# ---------------------------------------------------------------------------- #
+# Predicting options
+# ---------------------------------------------------------------------------- #
+_C.PREDICT = CfgNode()
+
+_C.PREDICT.ENABLE = True
+
+_C.PREDICT.LABEL_FILE_PATH = './tmp/kinetics_400_labels.csv'
```

Comment:
Maybe it is better to keep the default to be `""` to avoid breaking other use cases.
`_C.PREDICT.LABEL_FILE_PATH =  ""`